### PR TITLE
Add changes based on PSA Attestation API 1.0.1 (ARM IHI 0085)

### PR DESCRIPTION
**Pull request by adrianlshaw**
@thomas-fossati

### CODE REVIEW

File: psa-tokens/draft-tschofenig-rats-psa-token.md [hunk 1]
```diff
@@ -246,7 +245,7 @@ As noted, some fields must be at least 32 bytes long to provide sufficient crypt
 | -75006 | Software Components | arm_psa_sw_components  | Array of map entries (compound map claim). See below for allowed key-values. |
 | -75007 | No Software Measurements | arm_psa_no_sw_measurements  | Unsigned integer |
 | -75008 | Auth Challenge | arm_psa_nonce  | Byte string |
-| -75009 | Instance ID | arm_psa_UEID  | Byte string |
+| -75009 | Instance ID | arm_psa_UEID  | Byte string (the type byte of the UEID should be set to 0x01. The type byte is described in the EAT draft.) |
```
Comment (thomas-fossati) [inline psa-tokens/draft-tschofenig-rats-psa-token.md]:
```suggestion
| -75009 | Instance ID | arm_psa_UEID  | Byte string (the type byte of the UEID should be set to 0x01. The type byte is described in {{I-D.ietf-rats-eat}}.) |
```

Comment (thomas-fossati) [reply psa-tokens/draft-tschofenig-rats-psa-token.md]: I think we cannot be confident: the draft is by definition a work in progress.
So, the idea Hannes and I discussed is we do the necessary surgery to incorporate and freeze the parts we reference from EAT in a subsequent PR.
WDYT?

Comment (thomas-fossati) [reply psa-tokens/draft-tschofenig-rats-psa-token.md]: > I'm OK with handling that in a separate PR.

Cool, I'll work on that plan starting next week.

Comment (thomas-fossati) [reply psa-tokens/draft-tschofenig-rats-psa-token.md]: @adrianlshaw: #45 if you want to take a look

Comment (adrianlshaw) [reply psa-tokens/draft-tschofenig-rats-psa-token.md]: Thanks!

**Pull request merged**